src/coord/src/timestamp.rs

```diff
@@ -1306,6 +1306,8 @@ impl Timestamper {
             .set("max.poll.interval.ms", "300000") // 5 minutes
             .set("fetch.message.max.bytes", "134217728")
             .set("enable.sparse.connections", "true")
+            // should match the isolation level of the source; for now that's always read_committed
+            .set("isolation.level", "read_committed")
```

Comment:
could we make an issue to share the implementation of `create_kafka_config` here as in `dataflow::source`, assuming that that makes sense? It seems like we'd want changes in either place to be mirrored.